---
content_type: page
description: This section provides the schedule of lecture topics along with summaries
  of each lecture.
hide_download: true
hide_download_original: null
learning_resource_types:
- Lecture Notes
ocw_type: CourseSection
title: Lecture Summaries
uid: 6be8bec6-5fb2-41d4-6042-c806fa910534
---

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
SES #
{{< thclose >}}
{{< thopen >}}
TOPICS
{{< thclose >}}
{{< thopen >}}
LECTURE SUMMARIES
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Introduction" >}}{{< /anchor >}}Introduction
{{< tdclose >}}
{{< tdopen >}}


1.  Introduction of the instructors and students
2.  Course overview
3.  Using literature databases
4.  The review process: The journey from manuscript to published work
5.  Introduction of next week's papers


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Introduction_to_stem_cells_and_induced_pluripotency" >}}{{< /anchor >}}Introduction to stem cells and induced pluripotency
{{< tdclose >}}
{{< tdopen >}}


Cells located within the inner cell mass of the developing mammalian blastocyst have the ability to develop into all adult cell types. This ability is termed pluripotency. Grabarek et al. use fluorescence markers to follow the fate of these ICM cells as the floating blastocyst (pre-implantation) attaches to the uterine wall (post-implantation). Some of these ICM cells will segregate and become primitive endoderm. The fate of the primitive endoderm is quite restricted, as these cells are fated to become the yolk sac, an extra embryonic structure which provides nourishment to the rest of the embryo. Unexpectedly, the authors discover that the primitive endoderm precursor cells are more pluripotent than the rest of the inner cell mass cells. They propose that the primitive endoderm precursor cells are protected from environmental differentiation factors present in the developing embryo, allowing these cells to remain pluripotent for longer. This work illustrates the importance of environment and epigenetics in developmental fate during early development.

In 1962, John Gurdon asked the question of whether pluripotency is only possible during early development. He found that mature terminally differentiated cell nuclei contain the information necessary for pluripotency. Despite his pioneering work, many still believed that the process of differentiation was unidirectional and terminal. However, numerous studies now show that differentiated cells can be reprogrammed to a pluripotent state. In 2006, Takahashi and Yamanaka revealed that terminally differentiated cells can be reprogrammed into pluripotent cells by the introduction of a small number of transcription factors. The findings were deemed so important that Yamanaka and Gurdon received the 2012 Nobel Prize in Physiology or Medicine.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Epigenetic_Memory_and_Epigenetic_states" >}}{{< /anchor >}}Epigenetic Memory and Epigenetic states
{{< tdclose >}}
{{< tdopen >}}


Induced pluripotent stem cells (iPSCs) are pluripotent cells that have been reprogrammed from differentiated cells. No matter which cell type iPSCs are derived from (e.g. skin or neurons), they are able to form all of the cell types of an adult organism. Interestingly, early passage skin iPSCs favor differentiation into skin, while neuron iPSCs favor neuronal lineages (Kim et al.). Despite identical genetic material, differentiation is skewed depending on the donor cell type. This phenomenon is caused by an epigenetic memory of the donor cell. Epigenetics refers to heritable changes that occur in chromatin without altering the primary DNA sequence.

We now know that epigenetically, pluripotent stem cells can exist in many different states. For example, some pluripotent stem cells require the cytokine, LIF (Leukemia Inhibitory Factor) to maintain pluripotency, while others require the growth factor, FGF (fibroblast growth factor) to maintain pluripotency.

Scientists have designated pluripotent stem cells that require LIF as being in a "ground" epigenetic state. Those requiring FGF are designated to be in a "primed" epigenetic state. LIF and FGF activate distinct signaling pathways that converge on a core set of transcription factors (Oct4, Sox2, Myc, Nanog) required for pluripotency. Pluripotent stem cells in a "ground" epigenetic state can easily transition to a "primed" epigenetic state by the removal of LIF and addition of FGF. However, transitioning from the "primed" to "ground" state is much more difficult. Interestingly, while mouse embryonic stem cells are thought to be in a "ground" state, cultured human embryonic stem cells appear to exist in a "primed" state. Gafni et al. were able to convert human embryonic stem cells into the "ground" state by addition of a chemical cocktail.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "DNA_methylation" >}}{{< /anchor >}}DNA methylation
{{< tdclose >}}
{{< tdopen >}}


Methylation of CpG residues in DNA is one epigenetic mechanism regulating transcription. The establishment of DNA methylation is termed _de novo_ methylation and occurs during early development and gametogenesis.

In 1999, Okano et al. established that there are two _de novo_ methyltransferases that perform this function, Dnmt3a and Dnmt3b. When Dnmt3a and Dnmt3b were removed from embryonic stem cells, they observed a loss of _de novo_ methylation. Demethylation of DNA occurs during early development of primordial germ cells and is also required for reprogramming to the pluripotent state. Activation-induced cytidine deaminase (AID) has recently been discovered to play a critical role in this demethylation. Bhutani et al. discover a requirement for AID in reprogramming. They utilize an interesting reprogramming method involving the fusion of human fibroblasts and mouse embryonic stem cells.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "DNAmethyl-binding_proteins" >}}{{< /anchor >}}DNAmethyl-binding proteins
{{< tdclose >}}
{{< tdopen >}}
How do cells interpret DNA methylation patterns? The readers of methylated DNA are methyl-binding proteins that interact with other proteins to alter transcription rates. Today we will discuss the methyl-binding protein MeCP2. In 1998, Jones et al. discovered that MeCP2 recruits histone deacetylases to methylated DNA. Deacetylation of histones results in increased chromatin condensation (less access of RNA polymerase) and subsequent repression of transcription. Defects in MeCP2 have been linked to Rett Syndrome, a neurodevelopmental disorder that shares many characteristics with autism spectrum disorder. In this study Guy et al. utilize a mouse model of Rett Syndrome to show that many of the defects caused by loss of MeCP2 during development are reversible by reintroducing MeCP2 in adulthood. This finding is important, because it suggests that this disorder does not cause irreversible neurodevelopmental problems and gives us hope for medical intervention.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Histone_marks_and_epigenomic_sequencing_technologies" >}}{{< /anchor >}}Histone marks and epigenomic sequencing technologies
{{< tdclose >}}
{{< tdopen >}}


The accessibility of DNA to RNA polymerase can be altered via histone modifications at critical genomic locations. For example, trimethlyation of lysine 4 of the H3 histone (H3K4) is associated with higher DNA accessibility and subsequent transcriptional activation. Conversely, trimethylation of lysine 27 of the H3 histone (H3K27) is associated with lower DNA accessibility and subsequent transcriptional repression. Recent advances in sequencing technologies have made it possible to study these modifications on a genomic scale. Mikkelsen et al. and Bernstein et al. utilize a technique termed Chromatin Immunoprecipitation sequencing (ChIP-Seq). Using an antibody that binds to a particular histone modification, the authors collect DNA associated with that particular histone modification. The DNA is sequenced and aligned to the genome.

To learn more about this important technique, we have included an optional video from the Journal of Visualized Experiments. This online journal provides videos of scientists performing popular experimental techniques. In this particular video, Buro et al. describe how to perform ChIP using human cells.

Mikkelsen et al. use the ChIP technique to compare histone modifications in pluripotent and differentiated cell types. Using the same technique Bernstein et al. discover bivalent domains. These are regions that contain both a repressive histone mark (H3K27me3) and an activating histone mark (H3K4me3). The genes containing these marks are generally developmental genes that are silenced in pluripotent cells. Bernstein et al. hypothesize that these genes are poised for activation. The data from these and similar experiments have been uploaded to a [free public database (ENCODE)](https://genome.ucsc.edu/ENCODE/). 


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Polycomb_group_proteins" >}}{{< /anchor >}}Polycomb group proteins
{{< tdclose >}}
{{< tdopen >}}
How do cells interpret histone marks? Polycomb group proteins are one of many readers of histone modifications. Polycomb group proteins form complexes that recognize certain histone modifications and repress transcription. Boyer et al. were the first to analyze the co-occupancy of polycomb group proteins using genomic sequencing techniques. They found that these proteins are enriched at the promoters of developmentally regulated genes that have the repressive mark H3K27me3. EZH2 is the catalytic subunit of polycomb group protein 2. In addition to its role in differentiation, changes in EZH2 activity have been correlated with poor prognosis for many tumor types. Sneeringer et al. describe a point mutation found in EZH2 that is linked to B-cell lymphoma. Interestingly, the disease state requires individuals to be heterozygous at the EZH2 locus. The authors find that the point mutation in the EZH2 protein results in an enzyme that is less efficient in catalyzing the mono-methylation of H3K27 and more efficient in catalyzing the di- and tri-methylation of H3K27. The authors propose that this form of B-cell lymphoma requires the wild-type allele to catalyze H3K27me1 and the mutant enzyme to catalyze H3K27me2 and H3K27me3.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Enhancers" >}}{{< /anchor >}}Enhancers
{{< tdclose >}}
{{< tdopen >}}
Non-coding _cis_ regulatory elements play an essential role in coordinating gene expression networks during development. These elements were notoriously difficult to identify prior to recent advances in high- throughput sequencing technology. Visel et al. use chromatin immunoprecipitation of the histone acetyl transferase p300 followed by next generation sequencing to locate thousands of putative transcriptional enhancer elements in the mammalian genome. These technologies use miniaturized platforms for simultaneously sequencing up to 100 million short DNA pieces. Rada-Inglesias et al. take this a step further by performing ChIP-Seq on multiple histone modifications and p300 from embryonic cells. By evaluating a combination of modifications in this way they are able to classify enhancers into different chromatin states that correlate with varying levels of transcription.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Super_enhancers" >}}{{< /anchor >}}Super enhancers
{{< tdclose >}}
{{< tdopen >}}
Pluripotency requires high activity levels of the master transcription factors, Oct4, Sox2, and Nanog. Whyte et al. found that co-occupancy of these transcription factors and the co-activator complex, Mediator, are found at ES cell-specific enhancers (~100 bp). Some of these enhancers are clustered together in large genomic regions termed "super" enhancers (10-50 kb). Interestingly, Loven et al. was able to preferentially inhibit the activity of super enhancers by application of JQ1. JQ1 is a chemical that inhibits the coactivator Brd4. Loven et al. report that Brd4 is found at super enhancers and is involved in the activation of oncogenic genes, such as Myc.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Non-coding_RNA" >}}{{< /anchor >}}Non-coding RNA
{{< tdclose >}}
{{< tdopen >}}
Long ncRNAs (lncRNAs) comprise a rapidly expanding class of non-coding polyadenylated transcripts with emerging roles in gene regulation. Guttman et al. utilize state-of-the-art sequencing technologies to identify thousands of new mammalian lncRNAs. Rinn et al. transcriptionally analyzed the HOX loci, a region containing genes that are important for the establishment of the anterior-posterior body plan of the embryo. They identify a number of non-coding RNAs that exhibit differential expression along the developmental axes. One of these noncoding RNAs, termed HOTAIR, travels across chromosomes (_in trans_) from the HoxC locus to represses the HOXD cluster. HOTAIR is shown to be required for the occupancy of the polycomb repressive complex 2 at the HOXD locus.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Modeling_complex_biological_systems_Student_paper_discussions" >}}{{< /anchor >}}Modeling complex biological systems & Student paper discussions
{{< tdclose >}}
{{< tdopen >}}


Modeling disease pathology using human stem cells is a powerful alternative to current non-human disease models. Umbach et al. describe a novel culture condition to differentiate embryonic stem cells into functional motor neurons. When co-cultured at low density with muscle cells, embryonic stem cells will differentiate into motor neurons capable of forming neuromuscular synapses. With these types of models, researchers can use iPS cells from patients to study specific mutations in disease pathology.

Written assignment discussions: Students will introduce the abstract from their assignment and share their proposed experiments with the class.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Chromatin_nuclear_topology" >}}{{< /anchor >}}Chromatin nuclear topology
{{< tdclose >}}
{{< tdopen >}}
Chromatin organization within the nucleus is dynamic and non-random. During interphase, chromatin domains establish three-dimensional organization. This organization drives the transcriptional profile of the cell. Meister et al. use imaging techniques to assess subnuclear positioning of tissue-specific promoters during differentiation in c. _elegans_. The authors find no preferential nuclear position of these tissue specific promoter regions in undifferentiated cells. However, differentiation shifted these promoters to the nuclear lumen or the nuclear envelope. Localization to the lumen was associated with activation of the promoter, while nuclear envelope localization was associated with silencing of the promoter. Hou et al. employed a clever high-throughput sequencing technique to map the 3 dimensional chromatin structure of _drosophila_ cells. The technique, termed Hi-C, captures chromatin structure by the addition of a cross-linking agent, such as formaldehyde. The DNA that remains in contact is ligated together and sequenced. Using this technique, Hou et al. identify chromatin domains that contain sharp boundaries in regions of active chromatin.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
13
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Stem_Cell_Therapy" >}}{{< /anchor >}}Stem Cell Therapy
{{< tdclose >}}
{{< tdopen >}}
The ability to replace damaged tissue using one's own reprogrammed cells represents an exciting avenue of regenerative medicine. While barriers to the clinical use of stem cell therapy exist, science is inching ever closer towards this reality. In 2007, Hanna et al. were able to reprogram fibroblasts derived from a mouse's tail, use gene targeting to correct a human sickle hemoglobin mutation in the resulting iPSCs, differentiate these iPSCs into hematopoietic stem cells, and inject these cells into the same mice, thereby rescuing their sickle cell anemia. One of the main obstacles to the therapeutic use of iPS cells remains the possibility of causing cancer. Traditionally, reprogramming required the viral activation of transcription factors. Furthermore, injection of pluripotent stem cells into the body can cause the formation of teratomas, a type of tumor that contains tissue derived from all three germ lineages. In Tsuji et al., the authors attempt to identify certain lines of iPSCs as safe. Those termed safe did not form teratomas when injected into immunocompromised mice. They show that the "safe" iPS cells could be differentiated into neurospheres and injected into mouse brain and spinal cord. Mice injected with the "safe" neurospheres displayed recovery from spinal cord injury.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
14
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Final" >}}{{< /anchor >}}Final
{{< tdclose >}}
{{< tdopen >}}
Oral Presentations and discussion of the course.
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}